docs/getting-started/tutorials/table_api.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Table API"
+nav-id: tableapitutorials
+nav-title: 'Table API'
+nav-parent_id: apitutorials
+nav-pos: 1
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+Apache Flink offers a Table API as a unified, relational API for batch and stream processing, i.e., queries are executed with the same semantics on unbounded, real-time streams or bounded, recorded streams and produce the same results.
+The Table API in Flink is commonly used to ease the definition of data analytics, data pipelining, and ETL applications.
+
+* This will be replaced by the TOC
+{:toc}
+
+## What Are We Building? 
```

Comment:
It's a general good practice in technical writing to address the user as "you" and avoid using "we". Could the content be rephrased to accommodate this?

Reply:
I agree in general, but I was making the change and it did sound right. I think the reason why is I don't view this a technical document so much as a conversation. We are taking the user through a tour of a sample application, skipping most of the details in favor of high-level concepts. Does that make sense or do I sound crazy?

Reply:
Thinking of just my preference, I would agree (so no — not crazy). Thinking about making documentation consistent in general (and considering this documentation first and a conversation second), I think it would be appropriate to cut the "we".

Reply:
Ok, I think we're in agreement. It will take me a little bit to remove "we"'s in a way that makes sense. In the meantime, let me know if you have any other comments content-wise.